packages/react-sdk/src/component-utils/image.test.ts

```diff
@@ -0,0 +1,155 @@
+import { getImageAttributes } from "./image";
+
+describe("Image optimizations applied", () => {
+  test("width is number, create pixel density descriptor 'x'", () => {
+    const imgAttr = getImageAttributes({
+      optimize: true,
+      width: 100,
+      height: 100,
+      src: "https://webstudio.is/logo.webp",
+      srcSet: undefined,
+      sizes: undefined,
+      quality: 100,
+    });
+
+    expect(imgAttr).toMatchInlineSnapshot(`
+      Object {
+        "sizes": undefined,
+        "src": "/cdn-cgi/image/width=256,quality=100,format=auto/https://webstudio.is/logo.webp",
```

Comment:
is this correct? `format=auto/https://webstudio.is/logo.webp` between the options and the URL I would have expected a comma: `format=auto,https://webstudio.is/logo.webp` but you know better

Reply:
Its right `/`